[IMP] web: Make it more usable on mobiles with more Bootstrap

Yajo opened the pull request:
Remove lots of custom styling, rely on Bootstrap and make it finally
near to usable in phones.

Description of the issue/feature this PR addresses:

Make the new phone backend somehow usable.



Current behavior before PR:

![peek 26-09-2018 08-45](https://user-images.githubusercontent.com/973709/46081981-cab5cc80-c195-11e8-8398-62d67bc917a1.gif)

Awful.

Desired behavior after PR is merged:

Big screen:
![bignow](https://user-images.githubusercontent.com/973709/46082015-e28d5080-c195-11e8-87cb-ff5de2c2a21f.gif)

Small screen:
![mobilenow](https://user-images.githubusercontent.com/973709/46082022-e620d780-c195-11e8-9e85-ba8ee9b5b833.gif)


Still not perfect, but usable at least.


--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr
@Tecnativa

pedrobaeza commented:
@Yajo please target to saas-11.5, not master, as this is not going to land on v12, which is the aim.

Yajo commented:
Switched to saas-11.5 then, and rebased to fix some conflicts. Still there's a JS test failing. I'm still not happy with the behavior of the main apps menu in phones, but I don't know if I'll have time to finish it this week... :man_facepalming: 

In any case, it's still better than it was, indeed.

angelmoya commented:
Tested on runbot, nivel!

One question, I tried to group by, there os expanded by default? Can I see amounts on group by?

Yajo commented:
I don't understand... Could you put a screenshot? O pregunta en español si quieres... :grin:

angelmoya commented:
Si, porque con mi inglés no me entenderá nadie 😂, por lo menos en español nos enteramos nosotros

Primero probé la agrupación de pedidos de venta desde el chrome del movil, y me salía como una cabecera con el nombre del cliente pero no agrupaba.

Ahora he probado desde el pc, en chrome con la herrmienta de desarrollo para simular vista movil, te paso un vídeo:

https://drive.google.com/file/d/1FegOIKENLjmXt68XOSAQQbmrWYpW1k_D/view?usp=sharing

Yajo commented:
This happened to you because you used that tool.

Try this:

1. Go to the sales page.
1. Resize the page to phone scale.
1. Press F5.
1. Use the search > groupby feature *for mobiles*.

The search UI is different when loaded from a phone, but you have to press F5 after setting the mobile size to get it.

angelmoya commented:
Ok, I will check next week ok sprint code

El vie., 28 sept. 2018 10:13, Jairo Llopis <notifications@github.com>
escribió:

> This happened to you because you used that tool.
>
> Try this:
>
>    1. Go to the sales page.
>    2. Resize the page to phone scale.
>    3. Press F5.
>    4. Use the search > groupby feature *for mobiles*.
>
> The search UI is different when loaded from a phone, but you have to press
> F5 after setting the mobile size to get it.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/odoo/odoo/pull/27247#issuecomment-425358139>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AGcAhX3QpmSFmpo_0rAyzBFsflgIxH82ks5ufdohgaJpZM4W6oOq>
> .
>

yelizariev commented:
Shall it be targeted to 12.0 branch now?

yelizariev commented:
@Yajo does EE work with this update?

Yajo commented:
No idea...

davidtranhp commented:
Would this get merged?

Yenthe666 commented:
@mart-e & @antonylesuisse pretty please? 😉

davidtranhp commented:
Any news on this, please?

Yenthe666 commented:
I'm going to guess that this one has a little political silver lining, which explains why there seems to be a silence on this specific PR honestly.

davidtranhp commented:
This reminds me of the story of Mambo and Joomla. I was a fan of Mambo and had been taking my steps to Joomla before Mambo die.

pedrobaeza commented:
You have the OCA alternative: https://github.com/OCA/web/pull/1066

davidtranhp commented:
I still insist on Odoo. But I start thinking of contributing to OCA instead of Odoo. The history of Mambo and Joomla seems get back?

pedrobaeza commented:
Well, this is an ecosystem, and both OCA and Odoo are important. And inside Odoo, both community and enterprise part matter, each of them with their niche.

rafaelbn commented:
Hello @ged-odoo @aab-odoo @mge-odoo from [Odoo Merge Sprint Salle Hocaille](https://github.com/odoo/odoo/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+involves%3Atecnativa+) !!!

Please this is a really nice improvement! Could you please take a  look?

The community in contributing! And as Fabien say today What make great a company is the Community! 

:heart:

aab-odoo commented:
@pparidans @adr-odoo @res-odoo @rfr-odoo

adr-odoo commented:
I'm sorry, but we don't consider this as a bug because Odoo Community edition is not supposed to be responsive. See https://www.odoo.com/page/editions (Version mobile).

In the same way, we can't guarantee that something working now will also work in the future. At this time, it's too much work to make it both (Community + Enterprise) responsive.

Nor is it reasonable to merge these changes into a stable release.

Thanks to https://github.com/odoo/odoo/pull/38768 that make Community usable without having to make it responsive, we avoid to show something totally broken.

Yenthe666 commented:
Now who once made a promise that Odoo 12 would get the responsiveness and enterprise-like design again? 🤔 Still waiting.
